docs/samples/browser-plugin-meetings/app.js

```diff
@@ -3795,7 +3814,7 @@ function createMembersTable(members) {
 
   thead.appendChild(createHeadRow());
 
-  Object.entries(members).forEach(([key, value]) => {
+  Object.entries(members).forEach(([_, value]) => {
```

Comment:
What is the reason for changing this from `key` to `_`?

Reply:
Just did this because Ide showed that it's unused, `"_"` uses for this purposes